x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_modal/index.tsx

```diff
@@ -328,6 +328,7 @@ export const EditExceptionModal = memo(function EditExceptionModal({
                 ruleName={ruleName}
                 isOrDisabled
                 isAndDisabled={false}
+                osTypes={exceptionItem.os_types}
```

Comment:
It looks like we're not allowing users to edit the OS after the exception has been created. I think this is probably fine but I wonder we should display the OS in the Edit modal.

Reply:
yea, I'm purposely not allowing OS changes in the OS modal.  We can display the OS selected though, that's a good addition.  will add.

Reply:
@peluja1012 just added the Edit read only on OS.

![image](https://user-images.githubusercontent.com/56395104/116479858-56140480-a84e-11eb-9b52-302a153c1b03.png)